src/index.html

```diff
@@ -5,6 +5,7 @@
     <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>WHAT</title>
+    <link href="https://fonts.googleapis.com/css2?family=Libre+Baskerville:ital,wght@0,400;0,700;1,400&family=Libre+Franklin:ital,wght@0,200;0,300;0,400;0,500;1,200;1,300;1,400;1,500&display=swap" rel="stylesheet">
```

Comment:
Why we connect fonts here? Can we connect it in CSS?

Reply:
Do you mean @import  link in css file 
or @font-face CSS-rule with downloading the fonts?

Reply:
`@import`

ref: https://stackoverflow.com/questions/14676613/how-to-import-google-web-font-in-css-file

Reply:
Done